.github/workflows/test_integration.yaml

```diff
@@ -15,11 +15,11 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
-    runs-on: ${{ matrix.os }}
+  integration-tests:
```

Comment:
Maybe we should change job name also for e2e tests?

Reply:
Aligned everywhere to `tests`